lib/base/collection.js

```diff
@@ -818,9 +870,11 @@ const attributeMethods = ['groupBy', 'countBy', 'sortBy'];
 // Use attributes instead of properties.
 _.each(attributeMethods, function(method) {
   CollectionBase.prototype[method] = function(value, context) {
-    const iterator = _.isFunction(value) ? value : function(model) {
-      return model.get(value);
-    };
+    const iterator = _.isFunction(value)
+      ? value
+      : function(model) {
```

Comment:
Also definitely dislike the ternary conditional operator split into multiple lines. Is this avoidable? Although in this case these previous 3 lines could be improved.

I see that not all ternary operators are reformatted like that, so do you know what's the trigger for splitting them into multiple lines?